parser/CMakeLists.txt

```diff
@@ -13,8 +13,6 @@
 # 01.Oct.2016, Jack Smith <jack.smith@elektrobit.com>, Original Author
 #
 
-set (CMAKE_AUTOUIC ON)
```

Comment:
`cmake_minumum_required`, `CMAKE_AUTOUIC ON`, `CMAKE_AUTOMOC ON`, `CMAKE_VERBOSE_MAKEFILE FALSE` are already set from the top `CMakeLists.txt`.
The rest of the `CMakeLists.txt` are added as subdirectories, so it not necessary to write this again.